doc/sphinx/refman-preamble.rst

```diff
@@ -24,6 +24,7 @@
 .. |Ltac| replace:: `L`:sub:`tac`
 .. |ML| replace:: :smallcaps:`ML`
 .. |OCaml| replace:: :smallcaps:`OCaml`
+.. |GtkSourceView| replace:: :smallcaps:`GtkSourceView`
```

Comment:
Suggestion: since this replacement is only useful for a single chapter, put this at the top of the relevant chapter instead of putting it here. A few other chapters already do this.

Reply:
OK, done.